CHANGELOG.md

```diff
@@ -2,6 +2,69 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.104.0](https://github.com/returntocorp/semgrep/releases/tag/v0.104.0) - 2022-07-13
+
+### Added
+
+- Added a new feature that was described in CLI-123. This was done support our overarching goal to improve the tool. (cli-123)
+- `semgrep ci` will now not block builds on triage ignored issues (cli-162)
+- Add support to cli/scripts/compare.py for podman environments (compare-script-podman)
+- A new experimental 'extract' mode. This mode runs a Semgrep rule on a codebase
+  and "extracts" code from matches, treating it as a different language. This
+  allows users to supplement an existing set of rules, e.g., for JavaScript, by
+  writing additional rules to find JavaScript in non-JavaScript files, e.g.,
+  JavaScript contained in HTML or template files. While this is somewhat possible
+  with metavariable-pattern, this reduces the work from an M \* N problem to an M
+  - N one. (gh-4478)
```

Comment:
This was saying `M + N one` originally. After applying pre-commit fixes it now looks like an indented bullet point list item. No good.